src/17-infinite-discovery/03-artworks-mutli2vec.ts

```diff
@@ -0,0 +1,183 @@
+import weaviate, { generateUuid5 } from "weaviate-ts-client"
+import _ from "lodash"
+import { GravityArtwork } from "./types"
+import { deleteIfExists } from "system/weaviate"
+import dotenv from "dotenv"
+import { getArtworks, getImageDataForArtworks } from "./helpers"
+
+dotenv.config()
+
+const CLASS_NAME = "InfiniteDiscoveryArtworks"
+const BATCH_SIZE = 2
```

Comment:
If the batch size if too big, `nginx` will respond with a `413 request entity too large` when trying to insert the objects into `weaviate`.